eth2/types/src/beacon_state.rs

```diff
@@ -1155,3 +1157,46 @@ impl From<ArithError> for Error {
         Error::ArithError(e)
     }
 }
+
+#[cfg(feature = "arbitrary-fuzz")]
+impl<T: EthSpec> arbitrary::Arbitrary for BeaconState<T> {
```

Comment:
We could probably write a proc macro for this, but I think it's probably more important that we start fuzzing ASAP. We can come back for a proc macro, I think not having it will only be annoying but never dangerous.

Happy to leave as is :)

Reply:
I thought about doing some procedural macro's especially in the BLS as it's almost the same code for most of the types but as you said I think it was more important to start fuzzing.